app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -59,11 +73,11 @@ def pr_id
         params[:pr_id]
       end
 
-      def author_reputation_reason
-        return :'contributed_code/major' if params[:labels].include?('reputation/contributed_code/major')
-        return :'contributed_code/minor' if params[:labels].include?('reputation/contributed_code/minor')
+      def author_reputation_level
+        return :major if params[:labels].include?('reputation/contributed_code/major')
+        return :minor if params[:labels].include?('reputation/contributed_code/minor')
 
-        :'contributed_code/regular'
+        :regular
```

Comment:
@ErikSchierboom `reason` and `level` are now two different things. "Reason" is part of the object (I actually think we can probably get rid of it now, but I'm not 100% so I've left it in). "Level"  is passed in where appropriate.